client/src/shared/components/Tabs/tabs.tsx

```diff
@@ -0,0 +1,32 @@
+import React, { useState } from 'react';
+
+interface Props {
+  items: string[]
+  children: React.ReactNode
+}
+
+const Tabs: React.FC<Props> = ({ items }: Props) => {
+  const [activeTab, setActiveTab] = useState('');
+
```

Comment:
- please follow the design in figma.
expected design and behavior should be.
  - should be able to render the design of tabs and content
  - if i click `tab 1` it should render content for tab 1.
  - if i click `tab 2` it should render content for tab 2.
  - for the content. okay to show just _"content for tab 1", "content for tab 2"_